x/ownership/internal/types/msg.go

```diff
@@ -35,13 +35,13 @@ func (msg MsgWithdrawCoins) Type() string {
 // ValidateBasic runs stateless checks on the message.
 func (msg MsgWithdrawCoins) ValidateBasic() error {
 	if msg.Amount.IsAnyNegative() {
-		return sdkerrors.Wrap(sdkerrors.ErrInsufficientFunds, "price must be positive")
+		return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "price must be positive")
 	}
 	if msg.Owner.Empty() {
-		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, msg.Owner.String())
+		return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, msg.Owner.String())
```

Comment:
Why changing the invalid address error? It seems that this error give less context

Reply:
For simplicity I changed most of the error to `ErrInvalidRequest` but maybe that was too much.
I will take a look to see which error can be used to give more context.

Reply:
I will improve the error on msg in a next PR when I improve the msg fields ;)